apps/web/app/(app)/environments/[environmentId]/surveys/components/PreviewSurvey.tsx

```diff
@@ -26,6 +28,8 @@ interface PreviewSurveyProps {
   environment: TEnvironment;
   languageCode: string;
   onFileUpload: (file: File, config?: TUploadFileConfig) => Promise<string>;
+  membershipRole?: TMembershipRole;
+  setLocalProduct?: React.Dispatch<React.SetStateAction<TProduct>>;
```

Comment:
These will always be present right?

Reply:
PreviewSurvey.tsx is being used at other places as well ,like here **apps/web/app/(app)/environments/[environmentId]/surveys/templates/TemplateContainer.tsx**

i need these two only for my AddLogoHere file, that is why these two are optional.